client/src/components/IndexPage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+
+const IndexPage = props => {
```

Comment:
I would recommend renaming this component to something that is more descriptive of what it is an index page for, like `TripsList`